feature/challenge/src/main/java/com/hmh/hamyeonham/challenge/ChallengeFragment.kt

```diff
@@ -342,7 +342,7 @@ class ChallengeFragment : Fragment() {
         TwoButtonCommonDialog.newInstance(
             title = getString(R.string.delete_app_dialog_title, clickedAppNameToDialog),
             description = getString(R.string.delete_app_dialog_description),
-            confirmButtonText = getString(R.string.all_okay),
+            confirmButtonText = getString(R.string.all_delete),
```

Comment:
<string name="delete_app_dialog_title">정말 %s 을(를) 삭제하시겠어요?</string>
```suggestion
                <string name="delete_app_dialog_title">정말 %s 을(를)\n삭제하시겠어요?</string>

```

Reply:
\n 줄바꿈 추가할까여
앱이름 길어지면 어케되고 있나요? 자동으로 줄바꿈되나요??

Reply:
width를 제약 주면 될 듯합니다 현재는 maxLine 2라는게 정해져있는 것을 보니? 아마 그 때 논의하고 진행했던 것 같아서 저희가 /n 을 굳이 넣을 필요는 없을 것 같아요